infra/lambda/lambda_function.py

```
@@ -167,6 +167,81 @@ def lambda_handler(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
+                            raise ValueError("Job ai2_input missing or invalid")
+
+                        remediation_response = generate_remediation(ai2_input)
+                        is_blocked = bool(remediation_response.get("blocked"))
+                        store.update_job_status(
+                            job_id=job_id,
+                            status="blocked" if is_blocked else "completed",
+                            result=remediation_response,
+                        )
+                        results.append({"job_id": job_id, "status": "blocked" if is_blocked else "completed"})
+
+                    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)